packages/main/src/themes/Toolbar.css

```diff
@@ -39,10 +26,6 @@
     width: 100%;
 }
 
-.ui5-tb-item[design="Transparent"] {
```

Comment:
I think that this is related to the slotted items. Are we sure that we want to remove it?